packages/storybook/src/documentation/versioning.mdx

```diff
@@ -0,0 +1,6 @@
+import { Markdown, Meta } from "@storybook/blocks";
+import markdown from "../../../../SEMVER.md?raw";
+
+<Meta title="Developer/Versioning" />
```

Comment:
```suggestion
<Meta title="Start/Versioning" />
```

Of "Introductie" of iets dergelijks. Wat vind jij?

Reply:
Net besproken, laten het in Developer, hierin komen ook License, Copyright (rijkshuisstijlgebruik ed)